hack/development-environment/Vagrantfile

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Probably should have a README.md in this directory with some quick documentation. WDYT?

Reply:
on it ! :)